MNNparcellation/bin/MNNparcellation

```diff
@@ -29,119 +29,140 @@ from MNNparcellation.inc.CSParcellation import Parcellation as CSP
 
 
 def check_input(Arg):
+
     test = False
-    if Arg.save == None or Arg.tractograms == None or Arg.tract_name == None or \
```

Comment:
When checking for None, use "is None".  if Arg.save is None or Arg.tractograms is None or Arg.tract_name is None or